ios/RNTextSize.m

```diff
@@ -6,8 +6,8 @@
 #import <React/RCTUtils.h>
 #else
 #import "React/RCTConvert.h"   // Required when used as a Pod in a Swift project
-#import "React/RCTFont.h"
-#import "React/RCTUtils.h"
+#import <React/RCTFont.h>
+#import <React/RCTUtils.h>
```

Comment:
Can you revert these changes unless you're sure you need them? I'm not sure if this was me, but regardless, it doesn't hurt to revert